runtime/src/stake_account.rs

```diff
@@ -103,6 +103,12 @@ impl<T> From<StakeAccount<T>> for (AccountSharedData, StakeState) {
     }
 }
 
+impl<T> StakeAccount<T> {
+    pub fn account(&self) -> &AccountSharedData {
+        &self.account
+    }
+}
+
```

Comment:
`pub(crate)` instead of `pub`.

also, can you please move this to:
https://github.com/solana-labs/solana/blob/102fb9807/runtime/src/stake_account.rs#L37-L47
to keep this organized and not to scatter `impl`s?